synapse/handlers/e2e_keys.py

```diff
@@ -241,6 +259,7 @@ def query_local_devices(self, query):
                     r["unsigned"]["device_display_name"] = display_name
                 result_dict[user_id][device_id] = r
 
+        log_kv(results)
```

Comment:
(a) don't we want `result_dict`?
(b) shouldn't we `log_kv({"results": result_dict})` ?

Reply:
a) yes
b) It would be formatted nicely in jaeger if we passed the result_dict as I have (It converts nested dicts to strings.. :roll_eyes:). However, we don't have a message for context. I can either make a copy of the dict so that I can include a top level "message" kv pair but I don't know if the constant coppying would be too expensive, log it as you've indicated along with a message, or just leave it as it is. Thoughts?

Reply:
ok, leave it